index.html

```diff
@@ -4,8 +4,195 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <title>Iranian film week in Finland</title>
   </head>
-  <body></body>
+  <body>
+    <section class="mobile-menu">
+      <img src="assets/x.svg" alt="" onclick="hideMobileMenu()" id="x" />
+      <ul>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="./about.html">About</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="#program">Program</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="#see-program-btn">Join</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="./about.html#sponsor">Sponsor</a>
+        </li>
+        <li><hr /></li>
+        <li class="mobile-menu-items" onclick="hideMobileMenu()">
+          <a href="#news">News</a>
+        </li>
+        <li><hr /></li>
+      </ul>
+    </section>
```

Comment:
- [ ] Kindly make sure to wrap your navigation bar into the `nav` tag in order to correctly structure your HTML semantically. :wink: